src/pages/Recipes/RecipeCreation/RecipeCreation.jsx

```diff
@@ -13,9 +13,11 @@ const RecipeCreation = ({
   handleEditLinkClick,
   isError,
   isLoading,
+  edition,
 }) => (
   <form className={styles.recipeCreationForm} onSubmit={handleSubmit}>
-    <Text size="h3">Dodaj przepis</Text>
+    {!edition && <Text size="h3">Dodaj przepis</Text>}
+    {edition && <Text size="h3">Edytuj przepis</Text>}
```

Comment:
Alternatively - we could get the text based on condition outside JSX and then pass it as variable. We could avoid duplicating the `Text` component and only rely on changing its content, wdyt?
```suggestion
const title = edition ? "Edytuj przepis" : "Dodaj przepis"
// ...
<Text size="h3">{title}</Text>
```